.travis.yml

```diff
@@ -13,13 +13,15 @@ env:
   - TEST_SUITE=doxygen_documentation
 
 before_install:
-  - docker pull dalg24/clang-format
+  - docker pull dalg24/clang-format:breathe
 
 script:
+  - ci_env=`bash <(curl -s https://codecov.io/env)`
```

Comment:
No, it's fine, it just seemed like an unrelated change to me. Do I understand it right that you use that `TRAVIS` env var  just for `scope_donotinclude = ['doxygen']`? If yes, could you please add a comment to that point?